tests/PerformanceHarness/performance_test_basic.py

```diff
@@ -220,7 +220,7 @@ def __init__(self, testHelperConfig: TestHelperConfig=TestHelperConfig(), cluste
         self.nodeosLogPath = self.nodeosLogDir/f"node_{str(self.validationNodeId).zfill(2)}"/"stderr.txt"
 
         # Setup cluster and its wallet manager
-        self.walletMgr=WalletMgr(True)
+        self.walletMgr=WalletMgr(True, port=7899)
```

Comment:
Where is the server side setting for this?

Reply:
This sets up both sides. The default is 9899 which collides with nodeos ports if more than 21 validator nodes are configured.